vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -26,6 +31,18 @@ def get_services(inventory)
 
     def get_pods(inventory)
       process_collection(inventory["pod"], :container_groups) { |n| parse_pod(n) }
+      @data[:container_groups].each do |x|
+        @data_index.store_path(:container_groups, :by_namespace,
+                               x[:namespace], :by_name, x[:name], x)
```

Comment:
You don't need the intermediate `:by_name` key here.  You could call the outer key `:by_namespace_and_name`, and then just use the two values